README.md

```diff
@@ -6,6 +6,7 @@
 Table of Contents
 
 - [Container Principles](principles.md)
+- [Specification Policies](policy.md)
```

Comment:
On Tue, Jan 05, 2016 at 07:56:41AM -0800, Vincent Batts wrote:
> > @@ -6,6 +6,7 @@
> >  Table of Contents
> >  
> >  - [Container Principles](principles.md)
> > +- [Specification Policies](policy.md)
> 
> it's not so much policy, as a style, right?

Agreed.  Updated in c41523b → 408412b.